Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeLocale.cpp

```diff
@@ -498,60 +581,99 @@ static constexpr Array<Span<StringView const>, @size@> @name@ { {
     append_mapping("s_currencies"sv, "s_currencies_{}", locale_data.currencies, [](auto const& value) { return value.currencies; });
 
     generator.append(R"~~~(
+struct CanonicalLanguageID {
+    Unicode::LanguageID to_unicode_language_id() const
+    {
+        Unicode::LanguageID language_id {};
+
+        language_id.language = language.to_string();
+        if (!script.is_empty())
+            language_id.script = script.to_string();
+        if (!region.is_empty())
+            language_id.region = region.to_string();
+        for (size_t i = 0; i < variants_size; ++i)
```

Comment:
Let's `language_id.variants.ensure_capacity(variants_size);` before this loop to avoid excessive regrows and moves.